packages/eligibility/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@publicgoods-scripts/eligibility",
+  "version": "0.1.0",
+  "private": true,
+  "description": "Digital Public Goods Eligibility Form",
+  "main": "index.js",
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.12.0",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "bootstrap": "^4.6.0",
+    "prop-types": "^15.7.2",
+    "react": "^17.0.2",
```

Comment:
The following 4 packages: `react`, `react-bootstrap`, `react-dom`, `react-scripts` are in the parent `package.json` so they are not needed here.

Reply:
Resolved in commit bc66e7e